### DISCUSSION TITLE
Charged twice for Copilot subscription

### DISCUSSION THREAD

**Discussion by abler98** [community/community#69897] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

I have an issue with payment

>Your GitHub Copilot subscription has ended.

Today I saw this message and then another amount was charged from my credit card. Next payment date wasn't changed. Is this a bug?

![image](https://github.com/community/community/assets/5261122/03071ca9-dd1a-4bed-b6e0-5de831ff1dbf)
![image](https://github.com/community/community/assets/5261122/a4b54f5c-a5f3-4a19-96bd-9a78d06ae4f0)

**Comment by Akash1134** [comment, marked as answer]
Hi @abler98 ! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). Please open a ticket with them :)

**Reply by abler98** [reply]
Thank you!

**Comment by suhli** [comment]
same here and get no response

**Reply by alisondavidperez** [reply]
Hey @Akash1134 I already opened a ticket 4 weeks ago, I created this ticket https://support.github.com/ticket/3384073 and I haven't recieved any answer.

**Reply by suhli** [reply]
> Hey @Akash1134 I already opened a ticket 4 weeks ago, I created this ticket https://support.github.com/ticket/3384073 and I haven't recieved any answer.

same here,they just take your money and never care your experience,fucking stupid github
